cmd/rekor-server/app/root.go

```diff
@@ -88,6 +94,21 @@ func init() {
 	if GitVersion != "devel" {
 		return
 	}
+	log.Logger.Debugf("pprof enabled %v", enablePprof)
+	// Enable pprof
+	if enablePprof {
+		go func() {
+			mux := http.NewServeMux()
+
+			mux.HandleFunc("/debug/pprof/", pprof.Index)
+			mux.HandleFunc("/debug/pprof/{action}", pprof.Index)
+			mux.HandleFunc("/debug/pprof/symbol", pprof.Symbol)
+
+			if err := http.ListenAndServe(":6060", mux); err != nil {
```

Comment:
Not that I want to have an abundance of command line arguments, but should we make the IP and port configurable?

Reply:
I am on the fence. The command-line flags are growing a lot. It shouldn't become like this https://cortexmetrics.io/docs/configuration/arguments/ which is a project by itself IMO.

Reply:
we can always add it later if folks request it